en_US/LC_MESSAGES/django.po

```diff
@@ -2955,163 +2953,314 @@ msgstr "You have been unsubscribed from this notification."
+#: kitsune/products/jinja2/products/product.html:104
+#, fuzzy
+msgid "Thunderbird profiles"
+msgstr "Thunderbird"
+
+#: kitsune/products/jinja2/products/product.html:105
+#, fuzzy
+msgid "Migrate"
+msgstr "Migrate"
+
+#: kitsune/products/jinja2/products/product.html:106
+msgid "Themes"
+msgstr "Themes"
+
+#: kitsune/products/jinja2/products/product.html:126
+#, fuzzy
+msgid "Frequent Topics"
+msgstr "Topics:"
+
+#: kitsune/products/jinja2/products/product.html:127
+#, fuzzy
+msgid "Explore the knowledge base."
+msgstr "Search the Knowledge Base"
+
+#: kitsune/products/jinja2/products/product.html:140
+#, fuzzy
+msgid "Download Thunderbird"
+msgstr "Thunderbird"
+
+#: kitsune/products/jinja2/products/product.html:144 kitsune/sumo/jinja2/includes/common_macros.html:321 kitsune/wiki/jinja2/wiki/document.html:83
+#, fuzzy
+msgid "Systems and Languages"
+msgstr "Other Systems and Languages"
+
+#: kitsune/products/jinja2/products/product.html:145
+#, fuzzy
+msgid "Release Notes"
+msgstr "Legal Notices"
+
+#: kitsune/products/jinja2/products/product.html:146 kitsune/sumo/jinja2/base.html:326 kitsune/sumo/jinja2/includes/common_macros.html:323 kitsune/wiki/jinja2/wiki/document.html:85
+#, fuzzy
+msgid "Privacy"
+msgstr "Privacy Policy"
+
+#: kitsune/products/jinja2/products/products.html:3
+#, fuzzy
+msgid "Products"
+msgstr "Products"
+
+#: kitsune/products/jinja2/products/includes/topic_macros.html:15
+#, fuzzy
+msgid "See all"
+msgstr "Show media for:"
+
+#: kitsune/products/jinja2/products/includes/topic_macros.html:41
+#, fuzzy
+msgid "Still need help?"
+msgstr "Unreviewed Changes"
+
+#: kitsune/products/jinja2/products/includes/topic_macros.html:43
+msgid "We’re here for you. Post a question to our support forums and get answers from our community of experts."
+msgstr "We’re here for you. Post a question to our support forums and get answers from our community of experts."
+
+#: kitsune/products/jinja2/products/includes/topic_macros.html:50 kitsune/questions/jinja2/questions/question_list.html:103 kitsune/sumo/jinja2/includes/common_macros.html:48
+#, fuzzy
+msgid "Ask the Community"
+msgstr "How to Contribute"
+
+#: kitsune/products/jinja2/products/includes/topic_macros.html:62
+msgid "Contact Us"
+msgstr "Contact Us"
+
+#: kitsune/products/jinja2/products/includes/topic_macros.html:63
+msgid "Contact the Support Team for help."
+msgstr "Contact the Support Team for help."
+
+#: kitsune/products/jinja2/products/includes/topic_macros.html:64 kitsune/questions/jinja2/questions/new_question.html:5
+msgid "Ask a Question"
+msgstr "Ask a Question"
+
+#: kitsune/products/jinja2/products/includes/topic_macros.html:72 kitsune/questions/jinja2/questions/metrics.html:60 kitsune/search/forms.py:92
+#, fuzzy
+msgid "Topics"
+msgstr "Topics:"
+
+#: kitsune/products/jinja2/products/includes/topic_macros.html:120
+#, fuzzy
+msgid "All articles and threads"
+msgstr "All articles requiring review..."
```

Comment:
Should this be a real ellipses vs three periods? What is it not a period? What follows this?

Reply:
I am not sure where this is coming from. The template has the text in the msgid and the translation should be coming from the `msgstr` so the latter is the "translated" text. This is also marked as fuzzy here so maybe there is an issue with the translation.